test/install.spec.js

```diff
@@ -25,7 +25,11 @@ describe('install', function () {
         '--reporter=lcov',
         path.resolve(__dirname, '../src/cli/bin.js')
       ].concat(args), {
-        cwd: projectDirectory
+        cwd: projectDirectory,
+        env: {
+          ...process.env,
+          PATH: `${path.resolve(path.join(__dirname, '../node_modules/go-ipfs-dep/go-ipfs'))}:${process.env.PATH}`
```

Comment:
Good catch.  Better to use [`require.resolve`](https://nodejs.org/api/modules.html#modules_require_resolve_request_options) instead of hard-coding the path to the `node_modules` folder though.

Reply:
@achingbrain updated!